lib/asciidoctor-epub3/converter.rb

```diff
@@ -9,6 +9,7 @@ module Epub3
     # EPUB3 or KF8 publication file.
     class Converter
       include ::Asciidoctor::Converter
+      include ::Asciidoctor::Logging
```

Comment:
Interesting idea to encapsulate the shim. I should do that in Asciidoctor PDF. (No need to reply).